src/com/aca/homework/week4/json/processor/StringProcessor.java

```diff
@@ -0,0 +1,20 @@
+package com.aca.homework.week4.json.processor;
+
+public class StringProcessor {
+    private String json;
+    public StringProcessor(String json){
+        this.json = json;
+    }
+
+    public String getValue(String key){
+        int lastIndexOfKey = this.json.indexOf('"'+key)+key.length();
+        if (lastIndexOfKey < 0) return null; // if the key does not belong to the json string, return null
+        int firstIndexOfValue = lastIndexOfKey+5; // added 5 to the index to skip the :, ", and spaces
```

Comment:
why 5? what if we put some extra spaces?

Reply:
Just pushed the improved version, feel free to review it

Reply:
I tested the improved version for a json string containing spaces and it seems to be working